src/composables/useTransactions.ts

```diff
@@ -55,6 +55,8 @@ export type OrderReceipt = OrderMetaData;
 
 export type Transaction = {
   id: string;
+  originalId?: string;
```

Comment:
This will be handy when we have a modal that shows some more info about this transaction - we could link it to the previous one if we like.